src/CellData.jl

```diff
@@ -320,3 +320,35 @@ end
 CellData.jump(a::DistributedCellField) = DistributedCellField(map_parts(jump,a.fields))
 CellData.jump(a::SkeletonPair{<:DistributedCellField}) = a.⁺ + a.⁻
 CellData.mean(a::DistributedCellField) = DistributedCellField(map_parts(mean,a.fields))
+
+
+# DistributedCellDof
+
+struct DistributedCellDof{A,B} <: DistributedCellDatum
```

Comment:
I have the feeling that this type is not actually needed. 

I think we we may be able to implement the same functionality using `DistributedCellField` + multiple type dispatching on `B<:AbstractPData{<:CellDof}`

Note that the same approach is used for `FEFunction`s. There is no `DistributedFEFunction` type in GridapDistributed.

Reply:
Ok. I have realized that this comment does not apply. `CellDof` is not actually a `CellField`, thus it does not make sense to fit `DistributedCellDof` into `DistributedCellField`. Sorry about the hassle.

Therefore, yes, it seems that we need this type. On the other hand, why the argument `metadata` is required? Do you anticipate a future use-case where it might be needed?

Reply:
No, I just used another type as a template which had it. I can take it out.

Reply:
If we do not actually need it, better to take it out.

Reply:
Ok, I'll take it out.